portal/src/util/projectname.test.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * @jest-environment jsdom
+ */
+
+/* global describe, it, expect */
```

Comment:
Use the types from `@jest/globals`. See https://github.com/authgear/authgear-server/pull/2266/commits/5f22fcd77bd10f87e29d02014a9e903268319b16